src/main/java/Actions/ToolBar/OpenHubBrowserPanelAction.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * OpenBrowserPanelAction.java
+ *
+ * Copyright (c) 2015-2019 RHEA System S.A.
+ *
+ * Author: Sam Geren�, Alex Vorobiev, Nathanael Smiechowski 
+ *
+ * This file is part of CDP4-SDKJ Community Edition
```

Comment:
this probably needs to be a different statement

```This file is part of DEH-MDSYSML```

looks like all files have this header and need to be updated